src/booking/containers/booking-check/booking-check.component.spec.ts

```diff
@@ -11,15 +12,49 @@ describe('BookingCheckComponent', () => {
       declarations: [ BookingCheckComponent ]
     })
     .compileComponents();
-  }));
-
-  beforeEach(() => {
     fixture = TestBed.createComponent(BookingCheckComponent);
     component = fixture.componentInstance;
+    component.bookingProcess = {locationName: 'London'} as BookingProcess;
     fixture.detectChanges();
-  });
+  }));
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+  it('should create component and show the correct info message and values on row for today only case', () => {
```

Comment:
I recommend removing the repetitive DRY code here. If you want to check the caption, info message and location then this could be performed in one test.